### ISSUE TITLE
ts-loader / webpack 2

### ISSUE THREAD

**Issue by johnnyreilly** [aspnet/JavaScriptServices#785]
Hello!

I'm one of the maintainers of ts-loader.  Listening to the live asp net show I realised you're using it in your templates.  (Super cool BTW!)

There's 2 observations I'd make:

1. At present you're using the 0.8^ version of ts-loader.  I'd probably advise switching to the 1.x version if you're targeting webpack 1.  The "breaking changes" version change was only really about dropping official support for node 0.12 (which has now been end of lifed).  I'd be happy to submit a PR to do this if you'd like?

2. webpack 2 has now been released (around Christmas time).  It might be good to move to using webpack 2 in the templates.  ts-loader moved to 2.x for webpack 2.

**Comment by SteveSandersonMS** [comment]
Hey - thanks for your work on `ts-loader`!

I'm sorry to say we've moved most of the templates to `awesome-typescript-loader`, and will be moving the rest in due course. This is not because there was anything wrong with `ts-loader`, but simply because we measured `awesome-typescript-loader` to be significantly faster (about 2x). Maybe this is because of how it moves the type checking to a background thread so it can run in parallel with other aspects of the build - not sure of the exact mechanism. So, for those templates, we are already on Webpack 2, and will be migrating the others as we go.

Nonetheless, we did use `ts-loader` as the primary loader for quite a while, and it did a great job. So thanks again for your efforts with this!
